wordpress-mobile/WordPress-iOS
# Notification Likes list: show site row in user profile bottom sheet

Ref #16244 

This adds a new Site cell and displays it in the Site section of the User Profile bottom sheet. It's showing stub values for now. Fetching and showing real site info will be in a future PR. The bottom sheet should render correctly on iPhone in portrait and landscape.

To test:
- Enable the `newLikeNotifications` feature.
- Run on an iPhone. (It will show on an iPad, just not in the right place.)
- Go to Notifications > Likes and tap a notification.
- Tap a user.
  - Verify a Site row is displayed in the Site section.
  - Verify the sheet is partial height in portrait.
  - Verify the sheet is full height in landscape.
- Disable the `newLikeNotifications` feature.
- Go to Notifications > Likes and tap a notification.
  - Verify tapping the user opens the user's site.

---
| ![light_port](https://user-images.githubusercontent.com/1816888/114462870-e7be1980-9ba0-11eb-8d84-dff194c6d4c9.png) | ![dark_port](https://user-images.githubusercontent.com/1816888/114462888-ed1b6400-9ba0-11eb-88f4-015e6aa7c645.png) |
|--------|-------|
| ![light_land](https://user-images.githubusercontent.com/1816888/114462895-f0165480-9ba0-11eb-93fa-b54aad858afb.png) | ![dark_land](https://user-images.githubusercontent.com/1816888/114462903-f3114500-9ba0-11eb-9aed-35cb92ce3464.png) |

---

## Regression Notes
1. Potential unintended areas of impact
N/A. Feature is incomplete.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Feature is incomplete.

3. What automated tests I added (or what prevented me from doing so)
N/A. Feature is incomplete.

PR submission checklist:
